Data/src/main/java/com/sdk/data/repository/InPlayerAssetsRepositoryImpl.kt

```diff
@@ -36,4 +36,10 @@ class InPlayerAssetsRepositoryImpl(private val assetsRemote: AssetsRemote,
             list.map { mapAccessFee.mapFromModel(it) }
         }
     }
+    
+    override fun getAccessFeesv2(id: Int, voucher: Int): Single<List<AccessFeeEntity>> {
```

Comment:
Same here, voucher should be optional value

Reply:
Updated in newest commit